frontends/concrete-python/tests/mlir/test_converter.py

```diff
@@ -578,8 +581,8 @@ def assign(x, y):
 return %2
 
             """  # noqa: E501
-            if USE_MULTI_PRECISION
-            else """
+                if USE_MULTI_PRECISION
+                else """
```

Comment:
Could you manually indent these to their correct position if possible? Both might be supported but indented would be preferred!